src/Parser.php

```diff
@@ -495,10 +470,11 @@ protected function parseExamples()
         \assert(\array_key_exists('keyword', $token));
         $keyword = $token['keyword'];
         $tags = empty($this->tags) ? [] : $this->popTags();
+        ['title' => $title, 'description' => $description] = $this->parseTitleAndDescription($token);
```

Comment:
Note, this will actually produce a small behaviour change in legacy mode.

Previously, we ignored any title on an Examples: line (the `$token['value']` was parsed but just discarded). Control flow then moves to `parseTableRows` which only accepts `TableRow|Newline|Comment`.

Therefore if there was text on the line below `Examples:` this would have caused a ParserException due to the unexpected `Text` content.

Now, even in legacy mode, those lines will be read and the ExampleTableNode will be created with a (possibly multiline) name and a null description (the same as we parse Scenario: etc in legacy mode).

I think this is OK since any files with syntax like that would have been invalid until now?